config/services.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="Sylius\GmvBundle\Parser\DateParser" />
+        <service id="Sylius\GmvBundle\Parser\DateParserInterface" alias="Sylius\GmvBundle\Parser\DateParser" />
```

Comment:
I would be for not using FQCN (ref.: [proposed ADR](https://github.com/Sylius/Sylius/pull/15883))
```suggestion
        <service id="sylius_gmv.parser.date" />
        <service id="Sylius\GmvBundle\Parser\DateParserInterface" alias="sylius_gmv.parser.date" />
```
the same applies to other services